src/main/cpp/states/climber/ClimberManualState.cpp

```diff
@@ -76,20 +71,32 @@ void ClimberManualState::Init()
 
         m_climber->SetControlConstants(0, m_controlDataUpDown);
         m_climber->SetSecondaryControlConstants(0, m_controlDataRotate);
-        //m_climber->UpdateTargets(m_upDownMin, m_rotateMin);
-        m_climber->UpdateTargets(m_upDownMin, 0.0);
+        m_climber->UpdateTargets(m_upDownMin, m_rotateMin);
+        //m_climber->UpdateTargets(m_upDownMin, 0.0);
     }
 }
 
 
 void ClimberManualState::Run()           
 {
-    if (m_climber != nullptr && m_controller != nullptr && m_reach.get() != nullptr && m_rotate.get() != nullptr)
+    if (m_climber != nullptr && m_controller != nullptr )
     {
         auto armDownPercent = m_controller->GetAxisValue(TeleopControl::FUNCTION_IDENTIFIER::CLIMBER_MAN_DOWN);
         auto armUpPercent   = m_controller->GetAxisValue(TeleopControl::FUNCTION_IDENTIFIER::CLIMBER_MAN_UP);
         auto upDownPercent = armUpPercent - armDownPercent;
 
+        auto rotatePercent = m_controller->GetAxisValue(TeleopControl::FUNCTION_IDENTIFIER::CLIMBER_MAN_ROTATE);
+        if (rotatePercent > 0.0)
+        {
+            rotatePercent *= 0.50;
+        }
+        else
+        {
+            rotatePercent *= 1.0; //Negative is here to flip input

```

Comment:
What is the point of this?

Reply:
This makes going "backward" or going back to starting position faster then going "forward".  We had to flip the 1.0 to negative to make controlling for me easier.  The directions I just mentioned might be flipped, I can't remember off the top of my head.

Reply:
How does multiplying something by 1.0 change anything?

Reply:
Did something get lost?